package.json

```diff
@@ -86,6 +86,9 @@
   },
   "packageManager": "yarn@4.1.0",
   "lint-staged": {
-    "*.{js,mjs,cjs,ts,mts,cts}": "eslint --fix --report-unused-disable-directives"
+    "*": [
+      "eslint --fix --report-unused-disable-directives",
```

Comment:
should we keep the JS filter for eslint, but use `*` for prettier?

Reply:
I'd rather not. When using `format:eslint` we also go through all extensions and I would like to have the same behaviour happening for lint-staged

Reply:
True!